### PR TITLE
[Delivers #99048394] send report to budget office

### PR DESCRIPTION
**Pull request by phirefly**
For @afeld on https://www.pivotaltracker.com/story/show/99048394

### CODE REVIEW

**Comment by afeld** [comment]
Status update: messed around with this a bunch, and after a bunch of tries, _finally_ figured out a configuration that works well for Cloud Foundry. Some outstanding tasks it needs before merge:

* [ ] Take care of new inline `TODO`s inline
* [ ] Document the production setup
* [ ] Add tests for the report

**Comment by afeld** [comment]
Merging because it's running live in production, but would love a review anyway, which I can address in a follow-up PR.

File: config/clock.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+require 'newrelic_rpm'
+require 'clockwork'
+require_relative 'boot'
+require_relative 'environment'
+
+puts "Clockwork loaded."
+
+module Clockwork
+  if ENV['BUDGET_REPORT_RECIPIENT'] && ServerEnv.instance_index == 0
+    every(1.day, 'report_mailer.budget_status', at: '03:30', thread: true) do
```
Comment (afeld) [inline config/clock.rb]: @phirefly Hmm, you probably only wanted this sent on weekdays, right?

**Comment by phirefly** [comment]
No harm at this point. I will get more feedback about the reporting this week and create a story for any updates.

**Pull request merged**